aldryn_newsblog/tests/frontend/integration/pages/page.newsblog.crud.js

```diff
@@ -9,11 +9,49 @@
 // #############################################################################
 // INTEGRATION TEST PAGE OBJECT
 
-var newsblogPage = {
+var newsBlogPage = {
     site: 'http://127.0.0.1:8000/en/',
     mainElementsWaitTime: 12000,
     iframeWaitTime: 15000,
 
+    // log in, log out
+    editModeLink: element(by.css('.inner a[href="/?edit"]')),
+    usernameInput: element(by.id('id_cms-username')),
+    passwordInput: element(by.id('id_cms-password')),
+    loginButton: element(by.css('.cms_form-login input[type="submit"]')),
+    userMenus: element.all(by.css('.cms_toolbar-item-navigation > li > a')),
+    testLink: element(by.css('.selected a')),
+
+    cmsLogin: function (credentials) {
+        // object can contain username and password, if not set it will
+        // fallback to 'admin'
+        credentials = credentials ||
+            { username: 'admin', password: 'admin' };
+
+        newsBlogPage.usernameInput.clear();
```

Comment:
don't really like referencing parent object through a variable name, but i guess that is going to be adapted eventually when we move it to cms helpers